File: String/codeforces_41A.py

# Problem link ===>> https://codeforces.com/problemset/problem/41/A

def is_translation_correct(s, t):
  """Returns True if the translation is correct, False otherwise."""
  reversed_s = s[::-1]
  return reversed_s == t

s = input()
t = input()

if is_translation_correct(s, t):
  print("YES")
else:
  print("NO")

File: String/codeforces_41A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/41/A
// submission link ===>> https://codeforces.com/contest/41/submission/224950415

#include <iostream>
#include <algorithm>
#include <string>
using namespace std;

int main()
{
    string s, t;
    cin >> s >> t;

    reverse(s.begin(), s.end());

    if (s == t)
    {
        cout << "YES" << endl;
    }
    else
    {
        cout << "NO" << endl;
    }

    return 0;
}
